site/templates/meet.json.php

```diff
@@ -0,0 +1,42 @@
+<?php
+/**
+ * @var Kirby\Cms\Pages $upcoming
+ * @var Kirby\Cms\Pages $past
+ */
+$upcomingEvents = [];
+$pastEvents     = [];
+
+$since = param('since');
+
+foreach ($upcoming as $event) {
+	$upcomingEvents[] = [
+		'title'    => $event->shortTitle()->value(),
+		'date'     => $event->datetime()->format('c'),
+		'timezone' => $event->timezone()->value(),
+		'link'     => $event->link()->value(),
+		'isMeetup' => $event->isMeetup(),
+	
+	];
+}
+
+// Only meetups, no other events
+$past = $past->filterBy('city', '!=', '');
+
+// Use param since to filter meetups after given date
+if ($since = param('since')) {
+	$past = $past->filterBy('num', '>', $since);
+}
+
+foreach ($past as $event) {
+	$pastEvents[] = [
+		'title' => $event->shortTitle()->value(),
+		'date'  => $event->datetime()->format('c'),
+		'link'  => $event->link()->value(),
+		'isMeetup' => $event->isMeetup(),
+	];
+}
+
+echo json([
+	'upcoming' => $upcomingEvents,
+	'past'     => $pastEvents,
+]);
```

Comment:
`Suggestion`: If we reuse the `$upcoming` and `$past` variables instead of setting the arrays to `$*Events` variables, we could do:

```suggestion
echo json(compact('upcoming', 'past'));
```

Reply:
Not a fan of compact, so never use it